kubejs/server_scripts/crafting.js

```diff
@@ -396,19 +396,53 @@ ServerEvents.recipes((event) => {
 
     //Scanner
     event
-        .shaped("scannable:scanner", [" BL", "BCS", " BR"], {
+        .shaped("scannable:scanner", [" BL", "BPW", " BR"], {
             B: "create:brass_sheet",
             L: "create:redstone_link",
-            C: "createaddition:connector",
-            S: "createaddition:copper_spool",
+            P: "mmt:lunium_nova_plate",
+            W: "mmt:lunium_nova_wire",
             R: "thermal:cured_rubber",
         })
         .id("scannable:scanner");
     event
-        .shaped("scannable:blank_module", ["RBR", "RDR", " B "], {
+        .shaped("scannable:blank_module", ["RBR", "RDR", "WBW"], {
```

Comment:
it no longer looks like a usb thingy now that whole row grid is filled :cry:. maybe we just replace the redstone with lunium wire.

Reply:
hai